02-Workflow-Overview.Rmd

```diff
@@ -72,17 +73,26 @@ you!
 
 ### Creating a SymLink Folder
 
-As stated above, we tend to use SymLinks to shared OneDrive folders. To set up a
-`data` SymLink folder, from a OneDrive path that ends with `/data/`, you can use
-the following command in the terminal with the .Rproj open:
+As stated above, we tend to use SymLinks to shared OneDrive folders. To set up
+a `data` SymLink folder on a Mac, from a OneDrive path that ends with `/data/`,
+you can use the following command in the terminal with the .Rproj open:
 
-```         
+```
 ln -s ~/path/to/linked/folder/called/data/ .
 ```
 
 The `.` here means that you want that `data` folder to be called `data` as a
 subdirectory at the root of your current .Rproj.
 
+For Windows machines, you *do* have to state what you want the name of the new
+junction to be with the symlink. This should be done in a cmd terminal.
+
+```
+mklink /j "C:/path/to/folder/you/want/to/link" "new-symlink"
```

Comment:
I think that you have to specify the full file path of both the existing folder you want to link and where you want to put the new folder. So I would change this to something like:
`mklink /j "C:/path/to/put/new/linked/folder/data" "~/path/to/existing/OneDrive/folder/called/data"`

I would also specify that you cannot already have a folder called 'data/' in the directory you want to place the symlink folder (I forget if that is mentioned above or not).

Reply:
I agree with Caitlin!

Reply:
y'all are amazing. I think this existed in a previous version of the best practices, and so i was kinda making it up. :)